src/WebJobs.Script/Workers/Rpc/RpcInitializationService.cs

```diff
@@ -127,79 +98,28 @@ internal async Task InitializeRpcServerAsync()
             }
             catch (Exception grpcInitEx)
             {
-                var hostInitEx = new HostInitializationException($"Failed to start Rpc Server. Check if your app is hitting connection limits.", grpcInitEx);
+                throw new HostInitializationException($"Failed to start Rpc Server. Check if your app is hitting connection limits.", grpcInitEx);
```

Comment:
Interesting... were we ignoring this exception previously?

Reply:
I assumed this was an oversight and fixed it !

Reply:
@brettsam  - OK to resolve this comment?